### PR TITLE
Export SearchDropdown component

### PR DESCRIPTION
**Pull request by vasantvohra**

### CODE REVIEW

File: indico/web/client/js/exports.js [hunk 1]
```diff
@@ -46,5 +47,6 @@ window._IndicoReactI18n = IndicoReactI18n;
 window._IndicoReactUtil = IndicoReactUtil;
 window._IndicoUtilsAxios = IndicoUtilsAxios;
 window._IndicoUtilsDate = IndicoUtilsDate;
+window._IndicoUtilsCase = IndicoUtilsCase;
```
Comment (ThiefMaster) [inline indico/web/client/js/exports.js:50]: Why is this export needed? The code in that module looks simple enough to work fine without exporting it (importing JS without an export like the ones here just means the code gets duplicated in the plugin's JS bundle)

File: indico/web/client/js/react/components/index.js [hunk 2]
```diff
@@ -56,7 +56,7 @@ export {default as WTFPrincipalListField} from './WTFPrincipalListField';
 export {default as WTFPrincipalField} from './WTFPrincipalField';
 export {default as WTFLinkListField} from './WTFLinkListField';
 export {default as WTFSUIColorPicker} from './WTFSUIColorPicker';
-export {default as WTFSearchDropdown} from './WTFSearchDropdown';
+export {default as WTFSearchDropdown, SearchDropdown} from './WTFSearchDropdown';
```
Comment (ThiefMaster) [inline indico/web/client/js/react/components/index.js:59]: I guess importing it from `indico/react/components/WTFSearchDropdown` does not work?

Comment (SegiNyn) [reply indico/web/client/js/react/components/index.js:59]: We were getting a rather mixed behaviour importing this component, where we could import it like you said with no problems in one file but it gives an error in other places. I think it's related the bundles, for example the import works when used in a file injected in this bundle: `self.inject_bundle('index.js')` but it does not work when imported in a file for a specific WP e.g. `self.inject_bundle('registration-form.js', WPManageRegistration)`

File: indico/web/client/js/exports.js (hunk 1 above)
Comment (SegiNyn) [reply indico/web/client/js/exports.js:50]: We usually get an error when trying to import something that has not been exported, for this as well. Example error:
```
Uncaught TypeError: __webpack_require__.t is not a function
    at ./js/exports.js (exports.js:32:27)
    at __webpack_require__ (bootstrap:19:1)
    at __webpack_exec__ (exports.js:52:64)
    at exports.js:52:64
    at __webpack_require__.O (chunk loaded:23:1)
    at webpackJsonpCallback (jsonp chunk loading:44:1)
    at Array.forEach (<anonymous>)
    at jsonp chunk loading:48:1
    at jsonp chunk loading:49:1
    at startup:5:1
```

```
isArrayLikeObject.js:33 Uncaught TypeError: __webpack_require__.nmd is not a function
    at ../../../node_modules/lodash/isBuffer.js (isArrayLikeObject.js:33:1)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
    at ../../../node_modules/lodash/isEmpty.js (isEmpty.js:6:16)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
    at ../../../node_modules/flask-urls/lib/build.js (build.js:14:39)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
    at ./js/react/components/principals/PrincipalField.jsx (PermissionTree.jsx:88:42)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
    at ./js/react/components/principals/index.js (imperative.js:61:1)
    at __webpack_require__ (registration-form.03…9.bundle.js:1943:41)
```

**Pull request merged**